app/src/androidTest/java/com/android/sample/ui/listActivities/ListActivitiesTest.kt

```diff
@@ -23,6 +23,8 @@ import com.android.sample.ui.navigation.NavigationActions
 import com.android.sample.ui.navigation.Route
```

Comment:
All the changes of the dates are done so that these activties never expire, as they should not for the tests they are used in